src/features/culturalSurvey/CulturalSurveyCheckbox.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react'
+import LinearGradient from 'react-native-linear-gradient'
+import styled from 'styled-components/native'
+
+import { TouchableOpacity } from 'ui/components/TouchableOpacity'
+import { Exposition } from 'ui/svg/icons/categories/bicolor/Exposition'
+import { Validate } from 'ui/svg/icons/Validate'
+import { getSpacing, Typo } from 'ui/theme'
+
+type CulturalSurveyCheckboxProps = { title?: string; subtitle?: string }
+
+export const CulturalSurveyCheckbox = (props: CulturalSurveyCheckboxProps) => {
+  const [selected, setIsSelected] = useState(false)
+
+  // TODO : this should be replaced with what comes from backend data
+  const BicolorMuseumIcon = styled(Exposition).attrs(({ theme }) => ({
+    color: selected ? theme.colors.secondary : theme.colors.greyDark,
+    color2: selected ? theme.colors.primary : theme.colors.greyDark,
+  }))``
+
+  const StyledLinearGradient = styled(LinearGradient).attrs(({ theme }) => ({
+    colors: selected
+      ? [theme.colors.secondary, theme.colors.primary]
+      : [theme.colors.greyMedium, theme.colors.greyMedium],
+    useAngle: true,
+    angle: 175,
+  }))({ borderRadius: getSpacing(2) })
+
+  // TODO yorickeando : delete this once the component is connected to the backend data
+  const title = props.title ?? 'Visité un musée,'
+  const subtitle = props.subtitle ?? 'un monument, une exposition...'
+
+  return (
+    <StyledLinearGradient>
+      <QuestionContainer onPress={() => setIsSelected(!selected)} activeOpacity={0.95}>
+        <ActivityIconContainer>
+          <BicolorMuseumIcon />
+        </ActivityIconContainer>
+        <DescriptionContainer>
+          <Typo.ButtonText>{title}</Typo.ButtonText>
+          <GreyCaption>{subtitle}</GreyCaption>
+        </DescriptionContainer>
+        {selected ? (
+          <ValidateIconContainer>
+            <RedValidate />
+          </ValidateIconContainer>
+        ) : null}
```

Comment:
généralement, ailleurs dans l'app on utilise plutot cette syntaxe

```suggestion
        {selected && (
          <ValidateIconContainer>
            <RedValidate />
          </ValidateIconContainer>
        )}
```